### DISCUSSION TITLE
Input filterable date

### DISCUSSION THREAD

**Discussion by tovantiep** [laravel/nova-issues#5168] [Q&A] [answered]
- Laravel Version: 9.43.0
- Nova Version: 4.19.4
- PHP Version: 8.0.2

### Detailed steps to reproduce the issue on a fresh Nova installation:
I have entered a 6 digit year number but it is still valid, is there any way to disable the input?
![image](https://user-images.githubusercontent.com/74131799/208057869-67ac89ba-c15e-448e-baea-2b97f24284a2.png)

**Comment by crynobone** [comment, marked as answer]
The date input are handled directly by the browser and format is different depending on user locale. We can't reliably control this behavior
